roles/version_update_single_node/tasks/update_status_check.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Check update status on HC3
+  block:
+    - name: Increment retry_count
+      ansible.builtin.set_fact:
+        retry_count: "{{ 0 if retry_count is undefined else retries | int + 1 }}"
+
+    - name: Pause before checking update status - checks will report FAILED-RETRYING until update COMPLETE/TERMINATED
+      ansible.builtin.wait_for:
+        timeout: 60
+      delegate_to: localhost
```

Comment:
After you trigger update, status is first "FAILED-RETRYING", then it switches to normal value.
Do I understand correctly?

Can you remove this task ? The next task also waits on scale_computing.hypercore.version_update_status_info.
I think you can. I also think you know why you can't :). If unsure, just add comment, that maybe it can be removed (testing is pain here).

Reply:
As I am not entirely sure if by removing this task the role would still work, I've just added a comment to it (just to be careful, for the code works perfectly fine right now), as suggested.